lib/Frame.js

```diff
@@ -0,0 +1,41 @@
+var {Client} = require('./main.js')
+
+function Frame (ikey, skey, host, sig_version = 4, digestmod = 'sha512') {
```

Comment:
I don't think we need to let callers pass in both `sig_version` and `digestmod`. The `sig_version` should _imply_ which `digestmod` is being used. We wouldn't want to let callers shoot themselves in the foot by passing in a `sig_version` and `digestmod` mismatch and get weird errors.